src/platform/ESP32/BLEManagerImpl.h

```diff
@@ -199,12 +199,12 @@ class BLEManagerImpl final : public BLEManager,
     static constexpr uint32_t kAdvertiseTimeout     = CHIP_DEVICE_CONFIG_BLE_ADVERTISING_TIMEOUT;
     static constexpr uint32_t kFastAdvertiseTimeout = CHIP_DEVICE_CONFIG_BLE_ADVERTISING_INTERVAL_CHANGE_TIME;
     uint64_t mAdvertiseStartTime;
-    chip::Callback::Callback<> mAdvertiseTimerCallback;
-    chip::Callback::Callback<> mFastAdvertiseTimerCallback;
+    System::Layer::TimerCompleteFunct mAdvertiseTimerCallback;
```

Comment:
I think those member could be removed completely now that they're raw pointers. There's no point in allocating them in RAM.